### PR TITLE
ISSUES-40 Add getColumnTypeName support

### PR DESCRIPTION
**Pull request by sundy-li**

### CODE REVIEW

File: src/test/java/com/github/housepower/jdbc/QuerySimpleTypeITest.java [hunk 1]
```diff
@@ -119,4 +120,22 @@ public void apply(Connection connect) throws Exception {
...
+            public void apply(Connection connect) throws Exception {
+                Statement statement = connect.createStatement();
+                ResultSet rs = statement.executeQuery("SELECT number as a1, number + 1 as a2, number * 3 as a3 from numbers(1)");
+
+                Assert.assertTrue(rs.next());
+                ResultSetMetaData metaData = rs.getMetaData();
+                Assert.assertEquals(metaData.getColumnName(1), "a1");
+                Assert.assertEquals(metaData.getColumnTypeName(1), "UInt64");
+                Assert.assertEquals(metaData.getColumnName(2), "a2");
+                Assert.assertEquals(metaData.getColumnTypeName(2), "UInt64");
```
Comment (zhang2014) [inline src/test/java/com/github/housepower/jdbc/QuerySimpleTypeITest.java]: Assert.assertEquals(metaData.getColumnClassName(2), "java.lang.Long")

File: src/main/java/com/github/housepower/jdbc/data/type/complex/DataTypeNullable.java [hunk 2]
```diff
@@ -40,6 +40,11 @@ public Object defaultValue() {
         return nestedDataType.defaultValue();
     }
 
+    @Override
+    public Class javaTypeClass() {
+        return Object.class;
```
Comment (zhang2014) [inline src/main/java/com/github/housepower/jdbc/data/type/complex/DataTypeNullable.java]: nestDataType.javatypeClass

File: src/main/java/com/github/housepower/jdbc/data/type/complex/DataTypeTuple.java [hunk 3]
```diff
@@ -45,6 +45,11 @@ public Object defaultValue() {
         return new ClickHouseStruct("Tuple", attrs);
     }
 
+    @Override
+    public Class javaTypeClass() {
+        return Object[].class;
```
Comment (zhang2014) [inline src/main/java/com/github/housepower/jdbc/data/type/complex/DataTypeTuple.java]: java.sql.Struct.class

File: src/main/java/com/github/housepower/jdbc/data/type/complex/DataTypeEnum8.java [hunk 4]
```diff
@@ -41,6 +41,11 @@ public Object defaultValue() {
         return values[0];
     }
 
+    @Override
+    public Class javaTypeClass() {
+        return Enum.class;
```
Comment (zhang2014) [inline src/main/java/com/github/housepower/jdbc/data/type/complex/DataTypeEnum8.java]: different than the actual return value

File: src/main/java/com/github/housepower/jdbc/data/type/DataTypeFloat64.java [hunk 5]
```diff
@@ -29,6 +29,11 @@ public Object defaultValue() {
         return DEFAULT_VALUE;
     }
 
+    @Override
+    public Class javaTypeClass() {
+        return Float.class;
```
[the remark this answers is not in the document]
Comment (sundy-li) [reply src/main/java/com/github/housepower/jdbc/data/type/DataTypeFloat64.java]: Yeah

**Pull request merged**